src/js/base/NotFound.tsx

```diff
@@ -27,7 +24,9 @@ type NotFoundProps = {
 export function NotFound({ status = 404, message = "Not found" }: NotFoundProps) {
     return (
         <StyledNotFound>
-            <Label color="red">{status}</Label>
+            <Label className={`text-[${theme.fontSize.xxl}]`} color="red">
```

Comment:
Tailwind is (will be) our theme.

Make sure the `text-` suffix here is similar to what it was before in appearance. Like, you might have to use `xl` instead.
```suggestion
            <Label className="text-xxl" color="red">
```